### ISSUE TITLE
Scheduled jobs are executed twice.

### ISSUE THREAD

**Issue by chswtj** [rundeck/rundeck#1720]
[trigger.example.zip](https://github.com/rundeck/rundeck/files/156982/trigger.example.zip)

We have Rundeck 2.6.2-1 deployed as war file on Tomcat running behind Apache 2.2 on Red Hat Enterprise Linux Server release 6.5 (Santiago).
As DB we use Oracle DB 11gR2 on a second server.

On this setup we are currently executing approx. 4000 jobs per day on our production environment.

Since some days, we see, that a number of scheduled jobs are triggered twice within 1-60 seconds. i.e. a job is triggered and some seconds later again.
We did not see, that is was triggered 3 or more times. Only twice.

All scheduled jobs are marked as "Multiple Executions : NO"
Most of the scheduled jobs are scheduled to run once 1 day, the other ones are scheduled to run each 10 or 15 minutes.
Most of the jobs are running very short time, because they are trigger long running client jobs asynchronously.

In the attachment there is one of thees jobs, that is scheduled for every 10 minutes.
Furthermore there is a snippet of the rundeck log, where it is shown, that the job is triggered twice.

This is only 1 example for all the other jobs, described above.

**Comment by ahonor** [comment]
@chswtj Did you configure the instances to be in cluster mode? I think that will be necessary since they are both pointing to the same DB.

In rundeck-config.properties:
`rundeck.clusterMode.enabled=true`

In framework.properties:
`rundeck.server.uuid=<value generated by uuidgen>`

**Comment by chswtj** [comment]
Thanks for your fast reply.
I followed your advice and let the system run over the weekend.
No all jobs ran only once.
But I noticed, that on Saturday and Sunday night 2 jobs still ran twice.
I exported the jobs, removed them from rundeck and than reloaded them.
Now they seem to run only once as defined.

When I added the properties you mentioned above, I restarted rundeck.
But then now scheduled job was running. I had to reload all scheduled jobs again to get it run.
I hope that this is the normal behaviour.

Thanks again.
